src/datetime.rs

```diff
@@ -1420,7 +1420,7 @@ pub mod serde {
         where
             S: ser::Serializer,
         {
-            serializer.serialize_i64(dt.timestamp_nanos())
+            serializer.serialize_i64(dt.timestamp_nanos().unwrap())
```

Comment:
these unwraps move the panic from `DateTime::timestamp_nanos` to the serde modules.

Reply:
Good catch